src/store/indexeddb-local-backend.js

```diff
@@ -137,6 +137,24 @@ LocalIndexedDBStoreBackend.prototype = {
     },
 
     /**
+     * Having connected, load initial data from the database and prepare for use
+     */
+    init: function() {
```

Comment:
Is there *ever* a reason why we would want to `connect()` and NOT call `init()`? Can we bundle them together? Otherwise, it's not really clear which should come first: connect then init, or init then connect?